src/components/TitleBar/ViewTitleBar.js

```diff
@@ -92,15 +60,15 @@ class ViewTitleBar extends Component {
         const StarIcon = starred ? Star : StarBorder;
 
         return (
-            <Fragment>
+            <>
                 <div className={classes.titleBar}>
                     <span style={titleStyle}>{name}</span>
                     <div className={classes.actions}>
                         <div
                             className={classes.titleBarIcon}
                             onClick={onStarClick}
                         >
-                            <StarIcon className={classes.starIcon} />
+                            <StarIcon style={{ fill: colors.grey600 }} />
```

Comment:
className instead of inline styling?

Reply:
I had to do this one with the style property due to css priority - MUI was overriding the color.